scripts/spack/packages/umpire/package.py

```diff
@@ -183,6 +184,17 @@ def initconfig_hardware_entries(self):
                 arch_str = ",".join(archs)
                 entries.append(cmake_cache_string(
                     "HIP_HIPCC_FLAGS", '--amdgpu-target={0}'.format(arch_str)))
+            # BEGIN SERAC EDIT
```

Comment:
We may want to take out references to others codes as a general practice?

Reply:
Thanks for catching this.  Fixed in the last commit.